css/styles.css

```diff
@@ -126,10 +133,16 @@
 
 /* @group Column contents */
 
+.ab-column > ul {
+    padding: 0 20px 0 0;
```

Comment:
By removing this padding we would significantly reduce the gap between columns. Doing this without any reason except removing a tiny bit of CSS seems inappropriate to me.
(This was restored from 2d193ccc36be60025bd5d8d82f6238b0203b5ffb)